backends-velox/src/main/scala/io/glutenproject/execution/VeloxHashAggregateExecTransformer.scala

```diff
@@ -113,7 +113,7 @@ case class VeloxHashAggregateExecTransformer(
       RelBuilder.makeProjectRel(aggRel, expressionNodes, context, operatorId)
     } else {
       val extensionNode = ExtensionBuilder.makeAdvancedExtension(
-        Any.pack(TypeBuilder.makeStruct(getPartialAggOutTypes).toProtobuf))
+        Any.pack(TypeBuilder.makeStruct(false, getPartialAggOutTypes).toProtobuf))
```

Comment:
why it needs to set to false? @rui-mo  please check there is any problem when it's false.

Reply:
No, we are not using the nullability.

Reply:
ClickHouse backend cares about the nullability of complex types, because CH Array/Map/Struct types are not allowed to be wrapped with nullable.  

Since Velox doesn't care about the nullability, we decided to pass the actual nullable value when transforming Spark data type to substrait Type. And the plan will fallback especially for ClickHouse backend when met complex spark data types with nullable is true.